static/src/stylesheets/module/content-garnett/_types.scss

```diff
@@ -1273,3 +1273,16 @@ $quote-mark: 35px;
         min-height: 180px;
     }
 }
+// *************** Interactive elements ***************
+
+.interactive.element--showcase,
+.interactive.element--supporting {
+    background: $garnett-neutral-5;
+    padding-top: $gs-baseline/4;
```

Comment:
Is it necessary even without a border? Otherwise can it be moved into the `mq` below?

Reply:
@regiskuckaertz You are right, it doesn't need border 0. I have removed them.